redwood-treehouse-host/src/androidMain/kotlin/app/cash/redwood/treehouse/treehouseAppFactoryAndroid.kt

```diff
@@ -34,7 +36,7 @@ public fun TreehouseAppFactory(
   context: Context,
   httpClient: OkHttpClient,
   manifestVerifier: ManifestVerifier,
-  eventListener: EventListener = EventListener(),
+  eventListener: EventListener = defaultEventListener,
```

Comment:
An alternative is adding this to `TreehouseApp.kt`, but iOS app is not building right now, so adding it here to start with